models/user.model.js

```diff
@@ -61,32 +69,33 @@ UserSchema.methods = {
    * @param {String} password - input password
    */
   async passwordMatches(password) {
-    return bcrypt.compare(password, this.password)
+    return bcrypt.compare(password, this.password);
   },
-/**
- * Returns user object without password
- */
-  async transform() {
+  /**
+   * Returns user object without password
+   */
+  transform() {
     // add feilds to be selected
-    const fields = [
-      "id",
-      "name",
-      "email"
-    ];
-    return pick(fields, this)
-  }, 
-  generateToken() {
+    const fields = ['id', 'name', 'email'];
```

Comment:
The resembling logic in the [user.controller.js](https://github.com/Tyav/mentor-mentee/pull/4/files#r311534156) file  should be move or delete, modify this to suit your functionality and you are good.